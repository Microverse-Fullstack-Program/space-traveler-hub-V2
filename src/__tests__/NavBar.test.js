import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { Navbar } from 'react-bootstrap';
import setupStore from '../redux/store';

describe('NavBar snapshot', () => {
  test('should render heading correctly', () => {
    render(
      <Provider store={setupStore({})}>
        <Navbar />
      </Provider>,
    );
    const missions = document.querySelector('header');
    expect(missions).toMatchSnapshot();
  });
});
