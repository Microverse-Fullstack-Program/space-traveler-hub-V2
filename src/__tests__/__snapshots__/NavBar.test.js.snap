// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar snapshot should render heading correctly 1`] = `null`;
